RNMail/RNMail.m

```diff
@@ -110,6 +110,16 @@ + (BOOL)requiresMainQueueSetup
                 mimeType = @"application/zip";
             } else if ([attachmentType isEqualToString:@"text"]) {
                 mimeType = @"text/*";
+            } else if ([attachmentType isEqualToString:@"mp3"]) {
```

Comment:
at this point, it might be worth using a switch statement here. @adamski you wanna tackle that now?

i'm ok with shipping this as is too.

Reply:
Objective-C (like C) only supports integer values for switch statements, so not possible I'm afraid, although there are some [workarounds that can look similar](https://stackoverflow.com/questions/8161737/can-objective-c-switch-on-nsstring). I think its simpler to leave it as is.